Source/WebKit/WebProcess/GPU/media/RemoteAudioDestinationProxy.cpp

```diff
@@ -79,11 +79,9 @@ void RemoteAudioDestinationProxy::startRenderingThread()
             if (m_shouldStopThread)
                 break;
 
-            unsigned frameCount = WebCore::AudioUtilities::renderQuantumSize;
-            while (m_renderSemaphore.waitFor(0_s))
-                frameCount += WebCore::AudioUtilities::renderQuantumSize;
+            ASSERT(m_frameCount->size() == sizeof(uint32_t));
+            renderAudio(*static_cast<uint32_t*>(m_frameCount->data()));
```

Comment:
Not that even though you may now ask for a number of frames that is not a multiple of 128, our lower level code will STILL produce a number of samples that is a multiple of 128 (thus producing too many samples):
```
void AudioDestinationResampler::renderOnRenderingThreadIfPlaying(size_t framesToRender)
{
    if (!m_isPlaying)
        return;
    for (size_t pushedFrames = 0; pushedFrames < framesToRender; pushedFrames += AudioUtilities::renderQuantumSize) {
        if (m_resampler)
            m_resampler->process(m_renderBus.ptr(), AudioUtilities::renderQuantumSize);
        else
            callRenderCallback(nullptr, m_renderBus.ptr(), AudioUtilities::renderQuantumSize, m_outputTimestamp);
```

`AudioUtilities::renderQuantumSize` is 128.

Reply:
Maybe that's fine but it makes it less obvious what actually changed with this patch.